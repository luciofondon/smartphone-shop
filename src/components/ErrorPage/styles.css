.error-page {
    font-size: 40px;
    text-align: center;
    padding-top: 200px;
}

.redirect{
    color: blue
}