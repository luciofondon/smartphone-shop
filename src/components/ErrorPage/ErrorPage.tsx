import React from 'react';
import { Link } from 'react-router-dom';

import './styles.css';

function ErrorPage() {
  return (
    <div className="error-page">
      Pagina no existe
      <div className="redirect">
        <Link to={'/list'}>REDIRECCIONAR</Link>
      </div>
    </div>
  );
}

export default ErrorPage;
