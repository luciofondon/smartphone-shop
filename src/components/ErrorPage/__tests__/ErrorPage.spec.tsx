import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';

import ErrorPage from '../ErrorPage';

test('render ErrorPage', () => {
  const { container } = render(
    <Router>
      <ErrorPage />
    </Router>,
  );
  expect(container).toMatchSnapshot();
});
