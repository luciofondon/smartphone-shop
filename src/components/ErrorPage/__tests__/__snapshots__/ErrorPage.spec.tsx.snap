// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ErrorPage 1`] = `
<div>
  <div
    class="error-page"
  >
    Pagina no existe
    <div
      class="redirect"
    >
      <a
        href="/list"
      >
        REDIRECCIONAR
      </a>
    </div>
  </div>
</div>
`;
